### ISSUE TITLE
examples/v9.x.x not running

### ISSUE THREAD

**Issue by ripecosta** [i18next/react-i18next#1786]
## 🐛 Bug Report

I was doing some repo work when I came across this vulnerability report:
![image](https://github.com/user-attachments/assets/f405eee0-6553-4c7f-b248-f0c8dae53216)

When I tried to fix it I realised that none of the projects in `example/v9.x.x/` seem to run. They throw a `TypeError: fsevents is not a constructor` from chokidar.

`npm install` updates the lockfile but I wouldn't expect it to break.

## To Reproduce

- Clone the repo
- Run `cd examples/preact && npm install && npm run start`

## Expected behavior

I expected the examples to run.

## Your Environment

- *runtime version*: Node v20
- *i18next version*: 8.4.3
- *os*: Mac
- *npm version*: 10.7.0

**Comment by ripecosta** [comment]
Those examples seem to all be in the [v9.x.x branch](https://github.com/i18next/react-i18next/tree/v9.x.x). Should we just delete them from the master branch to get rid of the vulnerability alerts?

**Comment by adrai** [comment]
Yes, we can delete them, but maybe we need to change a link in the gitbook repo to target the old branch...

**Comment by ripecosta** [comment]
I see what you meant. It seems the v9 links were broken already because they are pointing to the new examples. I'll raise a PR
